src/pdu_loop/mod.rs

```diff
@@ -138,6 +138,17 @@ mod tests {
     use futures_lite::Future;
     use std::{sync::Arc, thread};
 
+    const MAX_TIMEOUT: crate::timer_factory::LabeledTimeout =
+        crate::timer_factory::LabeledTimeout {
+            duration: Duration::MAX,
+            kind: crate::timer_factory::TimeoutKind::Pdu,
+        };
+    const MIN_TIMEOUT: crate::timer_factory::LabeledTimeout =
+        crate::timer_factory::LabeledTimeout {
+            duration: Duration::ZERO,
+            kind: crate::timer_factory::TimeoutKind::Pdu,
+        };
+
```

Comment:
Considering these are used in a few places, but only in tests, would it make more sense to move them to `timer_factory.rs` and call them `TEST_MAX_TIMEOUT`/`TEST_MIN_TIMEOUT`?